{{cookiecutter.project_name}}/noxfile.py

```diff
@@ -1,20 +1,67 @@
 import nox
+from pathlib import Path
+import os
 
 # Options to modify nox behaviour
+nox.options.default_venv_backend = "uv|virtualenv"
 nox.options.reuse_existing_virtualenvs = True
 
+VENV_DIR = Path("./venv").resolve()
+
 @nox.session(name="docs")
-def build_docs(session):
+def build_docs(session: nox.Session) -> None:
     """Build the documentation and load it in a browser tab, rebuilding on changes."""
-    envbindir = session.bin
     session.install("-e", ".[docs]")
     with session.chdir("docs/"):
-        session.run(
-            "sphinx-autobuild",
-            "-j",
-            "auto",
-            "--open-browser",
-            "-qT",
-            ".",
-            f"{envbindir}/../tmp/html",
-        )
+        # For local development
+        if session.interactive:
+            session.run(
+                "sphinx-autobuild",
+                "-j",
```

Comment:
I think we'll either need to list `sphinx-autobuild` as a dependency in `pyproject.toml` or install it here using `session.install("sphinx-autobuild")`.

Reply:
It is already a dependency under `[project.optional-dependencies][docs]`, and we are doing `session.install( .. .[docs]` so I think we should be good and it works in CI.

Reply:
Never mind, I didn't notice – thanks!